File: Beecrowd/1 - Beginner/1000 - 1099/1048 - Salary Increase/C/1048_SalaryIncrease.c
#include <stdio.h>

int main() {
    double salary;
    scanf("%lf", &salary);

    double newSalary;
    int percentage;

    if (salary <= 400) {
        newSalary = salary * 1.15;
        percentage = 15;
    } else if (salary <= 800) {
        newSalary = salary * 1.12;
        percentage = 12;
    } else if (salary <= 1200) {
        newSalary = salary * 1.10;
        percentage = 10;
    } else if (salary <= 2000) {
        newSalary = salary * 1.07;
        percentage = 7;
    } else {
        newSalary = salary * 1.04;
        percentage = 4;
    }

    printf("Novo salario: %.2lf\n", newSalary);
    printf("Reajuste ganho: %.2lf\n", newSalary - salary);
    printf("Em percentual: %d %%\n", percentage);
    
    return 0;
}

File: Beecrowd/1 - Beginner/1000 - 1099/1048 - Salary Increase/1048_SalaryIncrease.py
salary = float(input())

new_salary = 0.0
percentage = 0

if salary <= 400:
    new_salary = salary * 1.15
    percentage = 15
elif salary <= 800:
    new_salary = salary * 1.12
    percentage = 12
elif salary <= 1200:
    new_salary = salary * 1.10
    percentage = 10
elif salary <= 2000:
    new_salary = salary * 1.07
    percentage = 7
else:
    new_salary = salary * 1.04
    percentage = 4

print(f"Novo salario: {new_salary:.2f}")
print(f"Reajuste ganho: {new_salary - salary:.2f}")
print(f"Em percentual: {percentage} %")
